04-single-hop-6lowpan-icmp/example_test_guide.md

```diff
@@ -0,0 +1,168 @@
+_The following is a possible way to run the tests, this is just an example_
+## Goal: Check single hop 6lowpan icmp
+
+_note: Since the ping6 function is now async some interval times may need to
+be increased to compensate_
+
+### Pre-setup Steps
+1. Get 2 of the following boards: `samr21-xpro`, `iotlab-m3`
+2. Get the following boards:`remote-revb`, `arduino-zero` with `xbee` module
+2. Go into the `cd ${RIOTBASE}/tests/gnrc_udp` directory
+2. Setup a terminal for each board _(tmux is your friend)_
+3. Find the serial numbers and ports with `ls /dev/serial/by-id/`
+4. For the samr21 terminals use the following to setup the env
+_eg_ `export BOARD=samr21-xpro && export SERIAL=<SAMR_SERIAL> && export PORT=<SAMR_PORT>`
+
+`export BOARD=samr21-xpro && export SERIAL=ATML2127031800008334 && export PORT=/dev/ttyACM0`
+
+`export BOARD=samr21-xpro && export PORT_=ATML2127031800002161 && export PORT=/dev/ttyACM1`
+5. For the remote-revb:
+_eg_ `export BOARD=remote-revb && export PORT=<REMOTE_PORT> && export PORT_LINUX=${PORT}`
+
+`export BOARD=remote-revb && export PORT=/dev/ttyUSB0 && export PORT_LINUX=${PORT}`
+
+`export BOARD=remote-revb && export PORT=/dev/ttyUSB1 && export PORT_LINUX=${PORT}`
+6. For the iotlab-m3 the flashing must be done when the other is unplugged as
+there is no way to distinguish between nodes:
```

Comment:
(one can also use the little toggle switch at the side of the board to turn them off/on ;-))

Reply:
Thanks I will add it.